core/src/main/java/org/apache/accumulo/core/metadata/schema/Ample.java

```diff
@@ -153,6 +153,15 @@ default TabletMetadata readTablet(KeyExtent extent, ColumnType... colsToFetch) {
   TabletMetadata readTablet(KeyExtent extent, ReadConsistency readConsistency,
       ColumnType... colsToFetch);
 
+  /**
+   * Entry point for reading multiple tablets' metadata. Generates a TabletsMetadata builder object
+   * and assigns the AmpleImpl client to that builder object. This allows readTablets() to be called
+   * from a ClientContext. Associated methods of the TabletsMetadata Builder class are used to
+   * generate the metadata.
+   */
+
+  TabletsMetadata.TableOptions readTablets();
```

Comment:
I wish we could enforce that there's no blank line between a method and its linked javadoc in the formatter, but I haven't found such a config.
```suggestion
   */
  TabletsMetadata.TableOptions readTablets();
```